### PR TITLE
[Merged by Bors] - spark-k8s bundle for 23.4

### PR DESCRIPTION
**Pull request by adwk67**
# Description

Bundle files for the 23.4 release of the spark-k8s operator.

Tests on Openshift 4.11:

```
--- PASS: kuttl (1097.71s)
    --- PASS: kuttl/harness (0.00s)
        --- PASS: kuttl/harness/pyspark-ny-public-s3-image_openshift-true_spark-3.3.0-stackable23.4.0_ny-tlc-report-0.1.0 (154.66s)
        --- PASS: kuttl/harness/resources_openshift-true_spark-3.3.0-stackable23.4.0_examples-3.3.0 (127.55s)
        --- PASS: kuttl/harness/spark-ny-public-s3_openshift-true_spark-3.3.0-stackable23.4.0_examples-3.3.0 (85.45s)
        --- PASS: kuttl/harness/logging_openshift-true_spark-3.3.0-stackable23.4.0_ny-tlc-report-0.1.0_examples-3.3.0 (279.31s)
        --- PASS: kuttl/harness/spark-examples_openshift-true_spark-3.3.0-stackable23.4.0_examples-3.3.0 (40.49s)
        --- PASS: kuttl/harness/spark-history-server_openshift-true_spark-3.3.0-stackable23.4.0_examples-3.3.0 (153.51s)
        --- PASS: kuttl/harness/spark-pi-private-s3_openshift-true_spark-3.3.0-stackable23.4.0_examples-3.3.0 (80.40s)
        --- PASS: kuttl/harness/spark-pi-public-s3_openshift-true_spark-3.3.0-stackable23.4.0_examples-3.3.0 (73.35s)
        --- PASS: kuttl/harness/pyspark-ny-public-s3_openshift-true_spark-3.3.0-stackable23.4.0 (92.54s)
```

## Definition of Done Checklist

- Not all of these items are applicable to all PRs, the author should update this template to only leave the boxes in that are relevant
- Please make sure all these things are done and tick the boxes
 
```[tasklist]
# Author
- [x] Changes are OpenShift compatible
- [x] Test cases tested on Openshift (4.11)
```

```[tasklist]
# Reviewer
- [ ] Changelog updated
```

```[tasklist]
# Acceptance
- [ ] Feature Tracker has been updated
- [ ] Proper release label has been added
```

Once the review is done, comment `bors r+` (or `bors merge`) to merge. [Further information](https://bors.tech/documentation/getting-started/#reviewing-pull-requests)

### CODE REVIEW

File: deploy/olm/23.4.0/manifests/stackable-spark-k8s-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+    olm.properties: '[]'
+    categories: Storage
+    capabilities: Full Lifecycle
+    description: Stackable Operator for Apache Spark
+    repository: https://github.com/stackabletech/spark-k8s-operator
+    containerImage: docker.stackable.tech/stackable/spark-k8s-operator:23.4.0
+
+  displayName: Stackable Operator for Apache Spark
+  description: |-
+    This is a Kubernetes operator to manage [Apache spark](https://spark.apache.org/) ensembles. The Stackable Apache Spark Operator
```
Comment (razvan) [inline deploy/olm/23.4.0/manifests/stackable-spark-k8s-operator.clusterserviceversion.yaml:18]:
```suggestion
    This is a Kubernetes operator to manage [Apache Spark](https://spark.apache.org/) ensembles. The Stackable Apache Spark Operator
```

File: deploy/olm/23.4.0/manifests/stackable-spark-k8s-operator.clusterserviceversion.yaml [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+    is part of the Stackable Data Platform, a curated selection of the best open source data apps like Kafka, Druid, Trino or Spark, all
+    working together seamlessly. Based on Kubernetes, it runs everywhere – on prem or in the cloud.
+
+    You can install the operator using [stackablectl or helm](https://docs.stackable.tech/spark-k8s/stable/getting_started/installation.html).
```
Comment (razvan) [inline deploy/olm/23.4.0/manifests/stackable-spark-k8s-operator.clusterserviceversion.yaml:22]: It just occurred to me that this sentence doesn't make sense when reading it in the OS marketplace :)

**Review by razvan — APPROVED** [review]
lgtm

File: deploy/olm/23.4.0/manifests/stackable-spark-k8s-operator.clusterserviceversion.yaml (hunk 2 above)
Comment (adwk67) [reply deploy/olm/23.4.0/manifests/stackable-spark-k8s-operator.clusterserviceversion.yaml:22]: I'll try and remember to remove it en route to the certification PR :)

**Comment by adwk67** [comment]
bors merge

**Pull request closed without merging**